impl/src/main/java/org/jboss/cdi/tck/tests/full/decorators/broken/custom/finalBeanClass/AfterBeanDiscoveryObserver.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.decorators.custom.broken.finalBeanClass;
+package org.jboss.cdi.tck.tests.full.decorators.broken.custom.finalBeanClass;
```

Comment:
Should be `full.decorators.custom.broken`, not `full.decorators.broken.custom`.

Reply:
Changed.